### ISSUE TITLE
Adding EPG in the new version would be great.

### ISSUE THREAD

**Issue by mikhoul** [freetuxtv/freetuxtv#231]
@SaiyanRiku   I've see that you are rewriting FreTuxTV in QT, IMO it would be so great to integrate [EPG](http://wiki.xmltv.org/index.php/XMLTVFormat) handling for TV channels .

Regards !  :octocat:

**Comment by forsaints** [comment]
I would like it too. It would be very good to input EPG for tv channels on FreeTuxTv.

**Comment by ski007** [comment]
EPG ! ...yes yes yes :-)

**Comment by forsaints** [comment]
Yes!! EPG in the version would be great. In addition it may add channels logos too.
